.github/workflows/publish.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
     - name: Use Node.js
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v3
       with:
-        node-version: 14
+        node-version: '16.20.1'
```

Comment:
this is keeping similar to build.yml workflow [here](https://github.com/Azure/vscode-bridge-to-kubernetes/blob/main/.github/workflows/build.yml#L23) and fix the failing publish/build to marketplace.